### DISCUSSION TITLE
After HA reboot often, the alexa media player is in error state and unusable

### DISCUSSION THREAD

**Discussion by Stephan-4711** [alandtse/alexa_media_player#2310] [General]
Hi,

often, when I restart HA the alexa media player doesn't load corretly. When I then restart AMP it's always working.
The only thing I found was:

Logger: homeassistant.config_entries
Quelle: config_entries.py:586
Erstmals aufgetreten: 06:56:04 (1 Vorkommnisse)
Zuletzt protokolliert: 06:56:04

Error setting up entry my@email.de - amazon.de for alexa_media
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 586, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/alexa_media/__init__.py", line 335, in async_setup_entry
    await login.login(cookies=await login.load_cookie())
                              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/alexapy/alexalogin.py", line 328, in load_cookie
    cookies = pickle.loads(await myfile.read())
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/cookies.py", line 312, in __setitem__
    raise CookieError("Invalid attribute %r" % (K,))
http.cookies.CookieError: Invalid attribute 'partitioned'

**Comment by nicedevil007** [comment]
same here again, sometimes this happens without any restart

**Reply by Stephan-4711** [reply]
Here I can reproduce it on almost every reboot. But the logs don't tell my why.

**Comment by ChristophCaina** [comment]
this

```
http.cookies.CookieError: Invalid attribute 'partitioned'
```

seems to be related to this:
https://gitlab.com/keatontaylor/alexapy/-/issues/29

and its origin seems to be within python 3.12 ... (until a workaround could be found / implemented)

**Reply by ChristophCaina** [reply]
PR #2380
newer alexapy version has removed this partitioned cookies....
